modules/react-map-gl-draw/src/editor.js

```diff
@@ -446,9 +446,7 @@ export default class Editor extends PureComponent<EditorProps, EditorState> {
       feature &&
       (this.props.mode === MODES.SELECT_FEATURE || this.props.mode === MODES.EDIT_VERTEX)
     ) {
-      this.props.onSelect({
-        selectedFeatureId: feature.id
-      });
+      this.props.onSelect({ selectedFeatureId: feature.id }, evt);
```

Comment:
If you mostly need the coordinates of the pointerdown position, How about

```
this.props.onSelect({
  selectedFeatureId: feature.id,
  mapCoords: [lat, lng],
  screenCoords: [x, y]
});
```

Reply:
I also need to understand whether the user right clicked on the current feature and by passing the mouse event i can do so. All clicks, including right ones, are going through the _onClick method, due to the anyclick bouding.

Reply:
Then could use the following format, which will be consistent with mode handling in `react-map-gl-draw` after refactor.

```
this.props.onSelect({
  selectedFeatureId: feature.id,
  mapCoords: [lat, lng],
  screenCoords: [x, y],
  sourceEvent: evt
});
```

Reply:
Yeah that works. I will update the diff